src/components/EmojiPicker/EmojiPickerButtonDropdown.tsx

```diff
@@ -67,14 +79,14 @@ function EmojiPickerButtonDropdown(props) {
                             style={styles.emojiPickerButtonDropdownIcon}
                             numberOfLines={1}
                         >
-                            {props.value || (
+                            {value ?? (
```

Comment:
I would leave `||` operator here, if for example `value` will be empty string (`''`) then `??` operator will select `''` as a value from this expression and that will be a bug